ntbs-service/Pages/ContactDetails/Edit.cshtml.cs

```diff
@@ -13,23 +13,26 @@ namespace ntbs_service.Pages.ContactDetails
 {
     public class EditModel : PageModel
     {
-        private readonly IUserService _userService;
         private readonly IUserRepository _userRepository;
         private readonly ValidationService _validationService;
+        private readonly UserHelper _userHelper;
 
-        public EditModel(IUserService userService, IUserRepository userRepository)
+        public EditModel(IUserRepository userRepository, UserHelper userHelper)
```

Comment:
It would be more consistent to inject an `IUserHelper` interface rather than the class itself. The main advantage of doing so is that it decouples the implementation of the `UserHelper` from this class... so for example we could easily replace the real `UserHelper` with a mock instance in our tests.